Common/include/CConfig.hpp

```diff
@@ -3539,25 +3539,25 @@ class CConfig {
    * \param[in] val_zone - Zone where the soler is applied.
    * \return Governing equation that we are solving.
    */
-  unsigned short GetKind_Solver(void) const { return Kind_Solver; }
+  ENUM_MAIN_SOLVER GetKind_Solver(void) const { return Kind_Solver; }
 
   /*!
    * \brief Governing equations of the flow (it can be different from the run time equation).
    * \param[in] val_zone - Zone where the soler is applied.
    * \return Governing equation that we are solving.
    */
-  void SetKind_Solver(unsigned short val_solver) { Kind_Solver = val_solver; }
+  //void SetKind_Solver(unsigned short val_solver) { Kind_Solver = val_solver; }
```

Comment:
```suggestion
  void SetKind_Solver(ENUM_MAIN_SOLVER val_solver) { Kind_Solver = val_solver; }
```
why comment this function?

Reply:
why have this function?

Reply:
It  indeed seems to be unused ... but then delete the function altogether instead of commenting it out